__tests__/promises.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+jest.mock('fs');
+
+const promises = require('../lib/promises.js');
```

Comment:
Again, `promises` is a very vague term, and it is close to the `Promise` class name. I would rename to something more descriptive